app/controllers/provider/admin/dashboards_controller.rb

```diff
@@ -19,7 +19,8 @@ def show
   end
 
   include DashboardTimeRange
-  helper_method :current_range, :previous_range, :backend_apis_presenter, :products_presenter
+  helper_method :current_range, :previous_range, :new_accounts_presenter, :potential_upgrades_presenter,
+    :backend_apis_presenter, :products_presenter
```

Comment:
May I suggest reordering these items so the changes are the fewest possible:
```suggestion
  helper_method :current_range, :previous_range, :backend_apis_presenter, :products_presenter,
    :new_accounts_presenter, :potential_upgrades_presenter,
```

Reply:
I'd prefer to leave as it is so that the order of the helper methods and the order of the method definitions are the same.